packages/nc-gui/components/smartsheet/SharedMapMarkerPopup.vue

```diff
@@ -0,0 +1,182 @@
+<script lang="ts" setup>
```

Comment:
in this file, it includes some gallery-related logic which is not required for map. please remove those code.

Reply:
done